installing/installing_nutanix/preparing-to-install-on-nutanix.adoc

```diff
@@ -1,12 +1,17 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="preparing-to-install-on-nutanix"]
-= Preparing to install on Nutanix
+= Installation methods
 include::_attributes/common-attributes.adoc[]
 :context: preparing-to-install-on-nutanix
 
 toc::[]
 
-Before you install an {product-title} cluster, be sure that your Nutanix environment meets the following requirements.
+You can install an {product-title} cluster on Nutanix using a variety of different installation methods. Each method has qualities that can make them more suitable for different use cases, such as installing a cluster in a disconnected environment or installing a cluster with minimal configuration and provisioning. Before you install {product-title}, ensure that your Nutanix environment meets the requirements.
+
+== Agent-based Installer
```

Comment:
All headings need a unique ID. And with the exception of the top level heading of an assembly like on line 3 in this file, they need a context at the end:
```suggestion
[id="preparing-to-install-on-nutanix-agent_{context}"]
== Agent-based Installer
```

Reply:
Should we include an example of what the install-config.yaml looks like for the Nutanix platform when using the agent-based installer?

Reply:
That might be helpful, though I'm not sure whether that would be better to add in the Nutanix section or in the Agent section of the install docs. 

Also, this addition roughly replicates the Agent blurb in the [vSphere section](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/installing_on_vmware_vsphere/preparing-to-install-on-vsphere#agent-based-installer), so anything we do here might be helpful to also add to those docs.

Reply:
@skopacz1 For adding the Nutanix agent-based installer install-config.yaml example, you may refer to the openshift/installer repo file cmd/openshift-install/testdata/agent/image/configurations/nutanix_compact.txt.